### PR TITLE
Adding tests for incorrect JWT context

### PR DESCRIPTION
**Pull request by adamstankiewicz**
## Description

Adding tests for incorrect JWT context (i.e., enterprise uuid).

## Post-review

Squash commits into discrete sets of changes

### CODE REVIEW

File: enterprise_catalog/apps/catalog/tests/test_rules.py [hunk 1]
```diff
@@ -55,6 +56,16 @@ def test_has_implicit_access_no_context(self, permission, system_wide_role, get_
         get_current_request_mock.return_value = self.get_request_with_jwt_cookie(system_wide_role)
         assert not self.user.has_perm(permission, TEST_ENTERPRISE_UUID)
 
+    @mock.patch('enterprise_catalog.apps.catalog.rules.crum.get_current_request')
+    @ddt.data(
+        ('catalog.has_admin_access', SYSTEM_ENTERPRISE_CATALOG_ADMIN_ROLE),
+        ('catalog.has_admin_access', SYSTEM_ENTERPRISE_OPERATOR_ROLE),
+    )
+    @ddt.unpack
+    def test_has_implicit_access_incorrent_context(self, permission, system_wide_role, get_current_request_mock):
```
Comment (bbaker6225) [inline enterprise_catalog/apps/catalog/tests/test_rules.py]: A small comment on what this is testing might be nice

**Review by bbaker6225 — APPROVED** [review]
Overall looks good. I think testing every method for the base EC serializer might be overkill now (I know I wrote those tests originally). I'd be fine with killing some of them off

**Review by christopappas — APPROVED** [review]

File: enterprise_catalog/apps/catalog/admin.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def get_catalog_query(self, obj):
 
 
 @admin.register(EnterpriseCatalogRoleAssignment)
-class EnterpriseDataRoleAssignmentAdmin(UnchangeableMixin, UserRoleAssignmentAdmin):
+class EnterpriseDataRoleAssignmentAdmin(UserRoleAssignmentAdmin):
```
Comment (adamstankiewicz) [inline enterprise_catalog/apps/catalog/admin.py:70]: Note: this class shouldn't inherit from `UnchangeableMixin`, which prevents making changes in django admin on stage.

**Comment by adamstankiewicz** [comment]
> Overall looks good. I think testing every method for the base EC serializer might be overkill now (I know I wrote those tests originally). I'd be fine with killing some of them off

Yeah, it might be overkill. Let's keep the tests for all the methods for now; we can decide to cut back on them later.

**Pull request merged**